tests/test.c

```diff
@@ -5180,12 +5194,15 @@ int test_can_create_topics(int skip) {
         return 0;
 #else
 
+        bootstrap = test_broker_version >= TEST_BRKVER(3, 0, 0, 0)
+                        ? "BROKERS" : "ZK_ADDRESS";
+
         if (!test_getenv("KAFKA_PATH", NULL) ||
-            !test_getenv("ZK_ADDRESS", NULL)) {
+            !test_getenv(bootstrap, NULL)) {
                 if (skip)
                         TEST_SKIP(
                             "Cannot create topics "
-                            "(set KAFKA_PATH and ZK_ADDRESS)\n");
+                            "(set KAFKA_PATH and BROKERS)\n");
```

Comment:
use %s  bootstrap here, as you did above.

Reply:
Thank you for the review, this has slipped. I have changed that and recommitted.